ldap-authenticator/src/main/java/org/xwiki/contrib/ldap/XWikiLDAPConnection.java

```diff
@@ -136,6 +137,14 @@ public LDAPConnection getConnection()
         return this.connection;
     }
 
+    /**
+     * @return the {@link LDAPConstraints} set for this connection.
+     */
+    public LDAPConstraints getConstraints()
```

Comment:
Not sure why you need this new API (needs a `@since` by the way), `getConnection().getConstraints()` seems enough to me.

Reply:
Indeed